Gemfile

```diff
@@ -44,9 +44,16 @@ gem "bootsnap", require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
 
+
+#pry
+gem 'pry', '~> 0.14.2'
```

Comment:
Use pry-rails gem in development and test only

Reply:
Added pry-rails instead of pry